test/scripts/e2e_subs/e2e-logs.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+filename=$(basename "$0")
+scriptname="${filename%.*}"
+date "+${scriptname} start %Y%m%d_%H%M%S"
+
+
+my_dir="$(dirname "$0")"
+source "$my_dir/rest.sh" "$@"
+function rest() {
+    curl -q -s -H "Authorization: Bearer $PUB_TOKEN" "$NET$1"
+}
+
+function app-txid {
```

Comment:
```suggestion
function app_txid {
```

people usually tend not to use hyphens in var/func names